packages/core/describers/transactions/lib/assetsTxDescribers.js

```diff
@@ -0,0 +1,256 @@
+import uuid from 'uuid/v1'
+import { findFunctionABI, TransactionDescriber } from '../TransactionDescriber'
+import { LogTxModel } from '../../../models'
+
+import {
+  TokenManagementInterfaceABI,
+  PlatformsManagerABI,
+  ChronoBankPlatformABI,
+} from '../../../dao/abi'
+import {
+  TX_CREATE_ASSET_WITHOUT_FEE,
+  TX_CREATE_ASSET_WITH_FEE,
+} from '../../../dao/constants/AssetsManagerDAO'
+import {
+  TX_CREATE_PLATFORM,
+  TX_ATTACH_PLATFORM,
+  TX_DETACH_PLATFORM,
+} from '../../../dao/constants/PlatformsManagerDAO'
+
+import {
+  TX_REISSUE_ASSET,
+  TX_REMOVE_ASSET_PART_OWNER,
+  TX_REVOKE_ASSET,
+} from '../../../dao/constants/ChronoBankPlatformDAO'
+
+export const FUNCTION_CREATE_ASSET_WITH_FEE = new TransactionDescriber(
+  findFunctionABI(TokenManagementInterfaceABI, TX_CREATE_ASSET_WITH_FEE),
+  ({ tx, block }, { address }) => {
+    address = address.toLowerCase()
+
+    if (tx.to.toLowerCase() === address || tx.from.toLowerCase() === address) {
+
+      const path = `tx.${TokenManagementInterfaceABI.contractName}.${TX_CREATE_ASSET_WITH_FEE}`
+      return new LogTxModel({
```

Comment:
Is it OK if function FUNCTION_CREATE_ASSET_WITH_FEE will not return anything? At the moment it will return new LogTxModel only 'if' at line 31 will be positive.